### DISCUSSION TITLE
能不能支持QQ频道

### DISCUSSION THREAD

**Discussion by fengyuecanzhu** [Arondight/Adachi-BOT#502] [Q&A] [answered]
如题，就是QQ刚出的频道功能，请问能不能支持呢

**Comment by mark9804** [comment, marked as answer]
要是作为消息收发基础的`oicq`库不支持频道功能的话（现在这个时间点好像确实没有支持），我们这里也没办法支持的

**Reply by Arondight** [reply]
https://github.com/takayama-lily/oicq/discussions/270

**Comment by mark9804** [comment]
[SDK 接入指南 | QQ机器人文档](https://bot.q.qq.com/wiki/develop/nodesdk/)
QQ 自己整了一个 `NodeSDK`，Silvery打算把频道和QQ bot拆开来开发，我们先看看 oicq 那边怎么安排吧

**Reply by fengyuecanzhu** [reply]
OK，了解
